app/helpers/money_formatting_helper.rb

```diff
@@ -0,0 +1,21 @@
+module MoneyFormattingHelper
+  def format_money_input(log:, question:)
+    value = log[question.id]
+
+    return unless value
+    return value unless question.prefix == "£"
+
+    number_with_precision(
+      value,
+      precision: 2,
+    )
+  end
+
+  def format_as_currency(num_string)
```

Comment:
a similar method already exists on `Log` - and precision: 2 is already the default value for that parameter.